### PR TITLE
✨ Rework of the embed page and Streameth logo on player

### PR DESCRIPTION
**Pull request by pblvrt**
I reworked the embed page so that it can receive session or stage id as well as the current playback id. If session or stage id are passed then the player will display the relevant thumbnail. 

I also added a Streameth logo to the player for branding

### CODE REVIEW

File: packages/app/app/[organization]/watch/page.tsx [hunk 1]
```diff
@@ -2,19 +2,19 @@ import { PlayerWithControls } from '@/components/ui/Player'
...
 import { fetchOrganization } from '@/lib/services/organizationService'
 import { Suspense } from 'react'
 import WatchGrid from '../components/WatchGrid'
 import { getVideoUrlAction } from '@/lib/actions/livepeer'
+import { generateThumbnailAction } from '@/lib/actions/sessions'
 const Loading = () => {
   return (
     <div className="flex flex-col gap-4 mx-auto w-full max-w-7xl h-full animate-pulse">
       <div className="flex flex-col w-full h-full md:p-4">
-        <div className="w-full bg-gray-300 aspect-video"></div>
```
Comment (pblvrt) [inline packages/app/app/[organization]/watch/page.tsx:17]: !

Comment (Mario-SO) [reply packages/app/app/[organization]/watch/page.tsx:17]: Could you be more descriptive? please

**Review by Mario-SO — APPROVED** [review]

**Comment by pblvrt** [comment]
closes #716

**Review by xannyxs — CHANGES_REQUESTED** [review]
@greatsamist will comment with a bug

**Comment by greatsamist** [comment]
Firstly, the player now displays the cover image on initial load, which is what we want. However, I'm uncertain about the player also displaying the cover image when the video is paused. Please take a look at the attached screenshot for reference.
image

**Comment by xannyxs** [comment]
> Firstly, the player now displays the cover image on initial load, which is what we want. However, I'm uncertain about the player also displaying the cover image when the video is paused. Please take a look at the attached screenshot for reference. image

It should show the frame it was paused at

**Review by greatsamist — APPROVED** [review]

**Review by greatsamist — APPROVED** [review]

**Comment by greatsamist** [comment]
@xannyxs  Can we now merge this PR? The requested changes have been addressed.

**Review by greatsamist — APPROVED** [review]

**Pull request merged**